index.html

```diff
@@ -6,14 +6,14 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio Website</title>
-    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="./styles.css">
 </head>
 
 <body>
     <header>
-        <div class="logo">Dhuruv</div>
+        <a class="logo">Dhuruv</a>
```

Comment:
- [ ] **[OPTIONAL]**

Great job implementing all the changes required by the previous reviewer. :clap: 
Your implementation was close, but here's a better way to achieve that :point_right: `<a class="logo" href="/">Dhuruv</a>`